kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/extensions/QaseExtension.java

```diff
@@ -62,52 +66,57 @@ public void executionStarted(TestIdentifier testIdentifier) {
 
     @Override
     public void executionFinished(TestIdentifier testIdentifier, TestExecutionResult testExecutionResult) {
-        if (!QaseClient.isEnabled() || !testIdentifier.isTest()
-                || !startTime.containsKey(testIdentifier)) {
-            return;
-        }
-        Duration duration = Duration.ofMillis(System.currentTimeMillis() - this.startTime.remove(testIdentifier));
         TestSource testSource = testIdentifier.getSource().orElse(null);
         Method testMethod = null;
         if (testSource instanceof MethodSource) {
             testMethod = getMethod((MethodSource) testSource);
         }
+        if (!testIdentifier.isTest() || !QaseClient.isEnabled()
```

Comment:
this whole if looks suspicious. Why do we create a test if qase is disabled or method is not a test?